[Server] Add eventType field to CTRLog entity to distinguish impression logs from click ones

Rwatana opened the pull request:
## Issue Number
#74 

## Background
In the frontend, retrieving only the `objectId` was necessary to determine which page element it belonged to. However, it was insufficient to distinguish between an impression and a click.  

To address this, we updated the table design and added a new `eventType` column. This allows us to separately track clicks and impressions, which is essential for calculating CTR (Click-Through Rate) by dividing the number of clicks by the number of impressions.

## Implementation Summary
Same as title.

## Test
![image](https://github.com/user-attachments/assets/560b058a-b5a5-47cc-a9eb-b516d1dae7d2)

## Schedule
2/25

Rwatana commented:
@nayuta-ai 
I updated the CTR log entity to match the front CTR property.
Please check my PR.

ryuji0123 commented on internal/server/domain/log.go:
```diff
@@ -12,8 +12,9 @@ type Log struct {
 }
 
 type CTRLog struct {
```
Add doc to explain each field.

nayuta-ai commented:
I think so too.
Please make the description clear about what was changed, such as **"Add `eventType` field to `CTRLog`"**.

ryuji0123 commented:
Also, there's no context in the PR description and @nayuta-ai  would have difficulty in understanding why we need eventType.

Rwatana commented:
@nayuta-ai 
- I added the PR background explaining why we need the `eventType` column.  
- I added documentation that explains the CTR log types and included example values.

ryuji0123 commented:
@Rwatana 
Title should be sth like:
[Server] Add eventType field to CTRLog entity to distinguish impression logs from click ones

Keep in mind titles should show what and why.

nayuta-ai commented on internal/server/domain/log.go:
```diff
@@ -11,9 +11,14 @@ type Log struct {
 type CTRLog struct {
-	CreatedAt time.Time
-	Objectid  string
+	EventType string    `json:"eventType"`
```
BTW, is JSON necessary here?
I feel like it's not needed since the domain is hidden from the presenter.

Rwatana replied:
I added it because I thought it would be easier to see, but I removed it because of the comments. I think so too.

Rwatana commented:
@nayuta-ai 
- I added doc for each field.
- I removed json.
Please check my code.
